cmd/test.go

```diff
@@ -384,17 +391,21 @@ func (t *Test) GetCmd() *cobra.Command {
 			}
 
 			path += "/keploy"
+			t.logger.Info("", zap.Any("keploy test and mock path", path))
```

Comment:
Why has this line been added

Reply:
Actually ```t.logger.Info("", zap.Any("keploy test and mock path", path), zap.Any("keploy testReport path", testReportPath))``` was a combined log and I have divided it into two different log for path and testReport Path.